packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.tsx

```diff
@@ -0,0 +1,70 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, navigate } from '@reach/router';
+import { Localized } from '@fluent/react';
+import { logViewEvent } from '../../lib/metrics';
+import { useNavigate } from '@reach/router';
+import Ready from '../../components/Ready';
+
+type ResetPasswordWithRecoveryKeyVerifiedProps = {
+  serviceName?: string;
+};
+
+const ResetPasswordWithRecoveryKeyVerified = ({
+  serviceName,
+}: ResetPasswordWithRecoveryKeyVerifiedProps & RouteComponentProps) => {
+  const navigate = useNavigate();
+  const viewName = 'settings.reset-password-with-recovery-key-verified';
+  const baseActionName = 'reset_password_with_recovery_key_verified';
+
+  // TO-DO: Attach correct ACTUAL actions to both the link and the button
+
+  return (
+    <>
+      <Ready
+        viewName={viewName}
+        serviceName={serviceName}
+        baseActionName={baseActionName}
+      />
+      <div className="flex justify-center mx-auto m-6">
+        <Localized id="reset-password-with-recovery-key-verified-generate-new-key">
+          <button
+            className="cta-primary cta-base-p font-bold mx-2 flex-1 p-2"
```

Comment:
I think the only classes we need here are `cta-primary cta-xl` and we'll have to move `cta-xl` into `fxa-react1.

Reply:
ohhhhh I'll try the cta-xl! Much more succinct 👍